web/src/components/VideoPlayer.jsx

```diff
@@ -93,7 +93,7 @@ export default function VideoPlayer({ children, options, seekOptions = {}, onRea
 
   return (
     <div data-vjs-player>
-      <video ref={playerRef} className="small-player video-js vjs-default-skin" controls playsinline />
+      <video ref={playerRef} className="small-player video-js vjs-default-skin" data-setup="{}" controls playsinline />
```

Comment:
Can you add a comment here so we know why setting `data-setup` to an empty object is required?

Reply:
Hopefully I did the comment correctly using the {/* */}. If needed I can try to dig up the link to the SO post that had that answer in there, but I have to be honest it took me quite a while to find out that was the issue lol